### PR TITLE
113 refactor into packages

### LINKED ISSUE #113: Refactor into packages

### PR DESCRIPTION
**Pull request by jluna28**
Does everyone agree with the class placements?

### CODE REVIEW

**Review by ARES-79 — APPROVED** [review]
I think its fine as long as the program still runs :)

File: src/main/java/Model/Blackboard.java [hunk 1]
```diff
@@ -261,9 +264,9 @@ private void writeUMLtoCodeQuestions(){
         boxes3.add(box8);
 
 
-        Hint hint1 = new Hint("Arrows with a solid black triangle head represent Association");
-        Hint hint2 = new Hint("Arrows with a outline triangle head represent Inheritance");
```
Comment (Mitashi-11) [inline src/main/java/Model/Blackboard.java]: This doesn't need to be Model.

File: src/main/java/Model/QuestionButtonsModel.java [hunk 2]
```diff
@@ -47,15 +49,15 @@ boolean submitPressed(String studentAttempt) {
         } else {
             String message = Blackboard.getBlackboard().getCurrentUser().getFirstName() + ", your answer is incorrect.";
             if (StringUtils.countMatches(Blackboard.getBlackboard().getCurrentQuestion().getAnswer(), "class") < StringUtils.countMatches(studentAttempt, "class")) {
-                message += "\nHint: You have made too many classes!";
+                message += "\nModel.Hint: You have made too many classes!";
```
Comment (Mitashi-11) [inline src/main/java/Model/QuestionButtonsModel.java]: Doesn't need Model.Hint

File: src/main/java/Model/User.java [hunk 3]
```diff
@@ -50,7 +52,7 @@ private int generateUserId(){
 
     @Override
     public String toString() {
-        return "User{" +
+        return "Model.User{" +
```
Comment (Mitashi-11) [inline src/main/java/Model/User.java]: ...

File: src/main/java/View/CodeToUMLPanel.java [hunk 4]
```diff
@@ -40,21 +47,21 @@ public CodeToUMLPanel(){
...
         MainController mC = new MainController(this, pairedText);
         DrawPanel east = new DrawPanel();
         east.setBackground(Color.LIGHT_GRAY);
-        Blackboard.getBlackboard().addObserver(east);
+        Blackboard.getBlackboard().addObserver((MyObserver) east);
         centerPanel.add(east, BorderLayout.CENTER);
 
         JToolBar selectionToolBar = new JToolBar();
-        JRadioButton association = new JRadioButton("Association");
-        JRadioButton inheritance = new JRadioButton("Inheritance");
```
Comment (Mitashi-11) [inline src/main/java/View/CodeToUMLPanel.java]: Here

File: src/main/java/View/CreateInstructorAccountPanel.java [hunk 5]
```diff
@@ -93,20 +97,19 @@ public void actionPerformed(ActionEvent e) {
...
             case ("Submit") ->{
-                //TODO: implement putting the data into the database
                 Instructor test = new Instructor(username.getText(), password.getText(),
                             firstName.getText(), lastName.getText(), null);
                 try {
                     Blackboard.getBlackboard().getDatabaseController().saveNewInstructor(test);
                     JOptionPane.showMessageDialog(this,
                             "Your class code is: "+test.getClassCode(),
-                            "New Instructor Class Code",
+                            "New Model.Instructor Class Code",
```
Comment (Mitashi-11) [inline src/main/java/View/CreateInstructorAccountPanel.java]: Maybe here

File: src/main/java/View/PracticePanel.java [hunk 6]
```diff
@@ -48,9 +54,9 @@ public PracticePanel(){
         centerPanel.add(east, BorderLayout.CENTER);
 
         JToolBar selectionToolBar = new JToolBar();
-        JRadioButton association = new JRadioButton("Association");
-        JRadioButton inheritance = new JRadioButton("Inheritance");
-        JRadioButton composition = new JRadioButton("Composition");
+        JRadioButton association = new JRadioButton("Model.Association");
+        JRadioButton inheritance = new JRadioButton("Model.Inheritance");
+        JRadioButton composition = new JRadioButton("Model.Composition");
```
Comment (Mitashi-11) [inline src/main/java/View/PracticePanel.java]: ....

File: src/main/java/View/QuestionPanel.java [hunk 7]
```diff
@@ -36,11 +41,11 @@ public QuestionButtonsModel getQuestionButtonsModel() {
      */
     void showHint(){
         if(hintIdx < Blackboard.getBlackboard().getCurrentQuestion().getHints().size()){
-            JOptionPane.showMessageDialog(this, Blackboard.getBlackboard().getCurrentQuestion().getHints().get(hintIdx).getText(), "Hint #" + (hintIdx + 1), JOptionPane.INFORMATION_MESSAGE);
+            JOptionPane.showMessageDialog(this, Blackboard.getBlackboard().getCurrentQuestion().getHints().get(hintIdx).getText(), "Model.Hint #" + (hintIdx + 1), JOptionPane.INFORMATION_MESSAGE);
```
Comment (Mitashi-11) [inline src/main/java/View/QuestionPanel.java]: Hint

File: src/main/java/View/UMLApp.java [hunk 8]
```diff
@@ -80,9 +85,9 @@ public UMLApp() {
         update.setSize(400,100);
 
         JPanel selectionPanel = new JPanel();
-        JRadioButton association = new JRadioButton("Association");
-        JRadioButton inheritance = new JRadioButton("Inheritance");
-        JRadioButton composition = new JRadioButton("Composition");
+        JRadioButton association = new JRadioButton("Model.Association");
+        JRadioButton inheritance = new JRadioButton("Model.Inheritance");
```
Comment (Mitashi-11) [inline src/main/java/View/UMLApp.java]: model.

File: src/main/java/Model/Blackboard.java (hunk 1 above)
Comment (Mitashi-11) [reply src/main/java/Model/Blackboard.java]: IT's the wrong line

**Pull request merged**